wheelbase: 2.67
steering_ratio: 18.61
wheel_radius: 0.31242 # 24.6in diameter tire 
